src/views/MarketsDropdown.tsx

```diff
@@ -173,9 +173,10 @@ const MarketsDropdownContent = ({
           >
             🇺🇸 {stringGetter({ key: STRING_KEYS.TRADE_US_PRESIDENTIAL_ELECTION })} →
           </Link>
-          <$IconButton
+          <IconButton
             onClick={() => setHasSeenElectionBannerTrupmWin(true)}
             iconName={IconName.Close}
+            fullIcon
```

Comment:
hmm the close button seems sorta big?

<img width="698" alt="Screenshot 2024-10-18 at 11 37 18 AM" src="https://github.com/user-attachments/assets/88892a3f-edf7-4959-95f0-a70b76b8311c">